install-and-configure/advanced-configuration/user-management-oidc/microsoft-entra-id-oidc-integration-for-kubecost.md

```diff
@@ -23,6 +23,7 @@ Before following this guide, ensure that:
 3. Select an appropriate name, and provide supported account types for your app.
 4. To configure `Redirect URI`, select _Web_ from the dropdown, then provide the URI as _https://{your-kubecost-address}/model/oidc/authorize_.
 5. Select _Register_ at the bottom of the page to finalize your changes.
+6. Do not add any Optional claims.
```

Comment:
Thanks for the update here @ivankube. However, do you think it would be better to move this down into the "Troubleshooting" section? We should also be specific about what we mean when we say "Optional Claims" for clarity.

It is specifically in the section "App Registrations -> Manage -> Token Configuration"

Reply:
Great suggestion, thank you @thomasvn!